panel/src/styles/utilities.css

```diff
@@ -0,0 +1,50 @@
+/* Align */
+[data-align="left"] {
+  --algin: start;
+}
+[data-align="center"] {
+  --algin: center;
+}
+[data-align="right"] {
+  --algin: end;
+}
+
+/* Invalid */
+[data-invalid] {
+  border: 1px solid var(--color-negative-outline);
+  box-shadow: var(--color-negative-outline) 0 0 3px 2px;
+}
+[data-invalid]:focus-within {
+  border: var(--field-input-invalid-focus-border) !important;
+  box-shadow: var(--color-negative-outline) 0 0 0 2px !important;
+}
+
+/* Tabbed */
+[data-tabbed] {
+  box-shadow: var(--shadow-outline);
+}
+
+/* Theme */
+[data-theme="positive"],
+[data-theme="success"] {
+  --theme: var(--color-positive);
+  --theme-l: var(--color-positive-light);
+  --theme-bg: var(--color-green-300);
+}
+
+[data-theme="negative"],
+[data-theme="error"] {
+  --theme: var(--color-negative);
+  --theme-l: var(--color-negative-light);
+  --theme-bg: var(--color-red-300);
+}
+[data-theme="notice"] {
+  --theme: var(--color-notice);
+  --theme-l: var(--color-notice-light);
+  --theme-bg: var(--color-orange-300);
+}
+[data-theme="info"] {
+  --theme: var(--color-focus);
+  --theme-l: var(--color-focus-light);
```

Comment:
TBH I would rename theme-l into theme-light or something like that. It's really hard to guess what it does.

Reply:
👍